addons/survey/tests/common.py

```diff
@@ -142,26 +148,35 @@ def assertResponse(self, response, status_code, text_bits=None):
     def _add_question(self, page, name, qtype, **kwargs):
         constr_mandatory = kwargs.pop('constr_mandatory', True)
         constr_error_msg = kwargs.pop('constr_error_msg', 'TestError')
+
+        sequence = kwargs.pop('sequence', False)
+        if not sequence:
+            sequence = page.question_ids[-1].sequence + 1 if page.question_ids else page.sequence + 1
+
         base_qvalues = {
-            'page_id': page.id,
-            'question': name,
+            'survey_id': kwargs.pop('survey', self.survey).id,
```

Comment:
Why not keeping explicitely giving survey_id=self.survey.id in calls to that method ? Kwargs update base values so nothing specific to do. Moreover we could even take it from the page.

Reply:
Done.